lambdas/credentialissuerreturn/src/test/java/uk/gov/di/ipv/core/credentialissuerreturn/CredentialIssuerReturnHandlerTest.java

```diff
@@ -72,7 +75,8 @@ void setUp() throws URISyntaxException {
     @Test
     void shouldReceive200AndJourneyResponseOnSuccessfulRequest() throws JsonProcessingException {
         CredentialIssuerReturnHandler handler =
-                new CredentialIssuerReturnHandler(credentialIssuerService, configurationService);
+                new CredentialIssuerReturnHandler(
+                        credentialIssuerService, configurationService, sqsService);
```

Comment:
Might be good to assert in this test that the sqsService has been called correctly?

Reply:
Yeah good shout 👍